HACKING.md

```diff
@@ -64,15 +64,17 @@ BUILDCMD="docker build" ./hack/build-image.sh
 
 # Running local builds with the installer
 
-If you want to run a local build with the installer, you need to do a bit of hacking. There is a script that will do most of the dirty work for you. It will modify the installer so the network operator doesn't run, and spin up a local copy of the network operator.
+If you want to run a local build with the installer, you need to do a bit of hacking. There is a script that will do most of the dirty work for you. It will modify the installer so the network operator doesn't run, and spin up a local copy of the network operator. This script will either override existing rendered cluster manifests ("override" mode), create a new cluster ("new" mode) and run openshift-install for you, or attach to an existing cluster ("attach" mode) and restart the network operator with overrides.
```

Comment:
So thinking about this some more...

One nice thing about the current system is that you can run `./hack/run-locally.sh` to bring up a cluster, do some stuff, figure out the bug you were investigating, hit `^C` to stop the CNO, edit some files, `./hack/build-go.sh`, and then `./hack/run-locally.sh` again to run the newly-rebuilt CNO against the already-running cluster.

I guess the new version still supports that, but you have to use different arguments the second time you run `./hack/run-locally.sh` to make it do `attach` rather than `new`. It would be cool if you didn't have to do that.